source/mqtt_client_connection.c

```diff
@@ -441,6 +441,52 @@ napi_value aws_napi_mqtt_client_connection_new(napi_env env, napi_callback_info
         }
     }
 
+    /* set reconnect min/max times. beware that user that might have passed only one of these values */
+    napi_value node_reconnect_min_sec = *arg++;
+    napi_value node_reconnect_max_sec = *arg++;
+    if (!aws_napi_is_null_or_undefined(env, node_reconnect_min_sec) ||
+        !aws_napi_is_null_or_undefined(env, node_reconnect_max_sec)) {
+
+        int64_t reconnect_min_sec = 1;
```

Comment:
If these are the internal (native) defaults, consider putting named constants in a shared part of the CRT and then using them as the defaults in the configuration objects.  Then these parameters should always be non-null and you can remove the constants here.